pyproject.toml

```
@@ -138,12 +138,12 @@ manylinux-x86_64-image = "manylinux2014"
 manylinux-i686-image = "manylinux2014"
 skip = "pp* cp36-* cp37-* *musllinux* *win32 *i686 cp38-macosx_arm64"
 test-skip = "*win32 *linux_i686"
-test-command = "python {project}/examples/python/stochastic_swap.py"
+test-command = "stestr run --abbreviate"
 # We need to use pre-built versions of Numpy and Scipy in the tests; they have a
 # tendency to crash if they're installed from source by `pip install`, and since
 # Numpy 1.22 there are no i686 wheels, so we force pip to use older ones without
 # restricting any dependencies that Numpy and Scipy might have.
-before-test = "pip install --only-binary=numpy,scipy numpy scipy"
+before-test = "pip install --only-binary=numpy,scipy numpy scipy | pip install -r requirements-dev.txt"
```

Is this syntax correct? I would expect this to pipe stdout from the first pip command into stdin of the second. I think you can just do:

```suggestion
before-test = "pip install --only-binary=numpy,scipy numpy scipy -r {project}/requirements-dev.txt"
```

---

yes @mtreinish totally right, I mistyped | instead of && for running this command sequentially. I was also wondering whether should we install everything on requirements-dev.txt or just stestr in aims of efficiency, but didn't have the time to check if the tests used all of these dev-dependencies.